test/lit/passes/string-lowering-instructions.wast

```diff
@@ -1,6 +1,6 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
 
-;; RUN: foreach %s %t wasm-opt -all --preserve-type-order --string-lowering -S -o - | filecheck %s
+;; RUN: wasm-opt %s -all --preserve-type-order --string-lowering -S -o - | filecheck %s
```

Comment:
Is there any harm to using foreach with a single module? it's future-proof...

Reply:
The only downside is that it makes it slightly more annoying to run the test manually, e.g. to repro bugs. Not much of a downside, but also a very easy change to make or undo, so I think it's worth it.